src/main/java/com/ethlo/clackshack/ClackShack.java

```diff
@@ -72,14 +72,12 @@ default ResultSet query(final String query, final Map<String, Object> params, fi
     /**
      * Kill the query with the specified query ID
      *
-     * @param queryId the id of the query to kill
-     * @return True if the query was found, false if it did not exist
+     * @param queryId     the id of the query to kill
+     * @param synchronous Whether to wait for the kill to finish
      */
-    default boolean killQuery(String queryId)
```

Comment:
Do you mind keeping this signature available with a default boolean value?

Reply:
Sure thing